.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test: --platform vm
+  # No parallelism on Firefox (-j 1)
+  # Causes flakiness – need to investigate
+  - test: --platform firefox -j 1
```

Comment:
Why Firefox? I thought most of our other packages used Chrome.

The tests on this package are fast enough that it's probably more efficient to run browser and VM tests in the same job.